2024-02-monthly-report.org

```diff
@@ -0,0 +1,203 @@
+
+A couple issues remain around the parsing of the ~format_args!()~ arguments, but we are expecting to take care of them soon. We will have to polish the integration of the Rust component responsible for parsing the ~format_args!()~ invocation, as there are build system subtleties to take in account.
+
+We will soon get in touch with the GCC release maintainers in order to prepare
+this upcoming release, with the expectation of our frontend being part of GCC 14.1.
+
+We would like to remind our readers that [[https://github.com/jdupak][Jakub Dupak]], one of our contributors working on
+integrating the ~polonius~ borrow-checker to gccrs, is currently finishing studying for his
+Master's Degree and is looking for sponsorship to continue his work on the borrow checker.
+Please get in touch if you or your company would be interested in funding his work!
+
+*** Community call
+
+We will have our next monthly community call on the 18th of March at 10am UTC. You can subscribe to our calendar
+to see when the next one will be held. The call is open to everyone, even if you would just
+like to sit-in and listen. You can also subscribe to our [[https://gcc.gnu.org/mailman/listinfo/gcc-rust][mailing-list]] or join our [[https://gcc-rust.zulipchat.com][Zulip chat]] to
+be notified of upcoming events.
+
+- [[https://meet.jit.si/gccrs-community-call-march][Jitsi link]]
+- Calendar ID: 7060a0923ffebd3cb52b1afef35a28ff7b64f05962c9af84c23b1847f1f5f894@group.calendar.google.com
+- [[https://calendar.google.com/calendar/embed?src=7060a0923ffebd3cb52b1afef35a28ff7b64f05962c9af84c23b1847f1f5f894%40group.calendar.google.com][Google calendar link]]
+- [[https://calendar.google.com/calendar/ical/7060a0923ffebd3cb52b1afef35a28ff7b64f05962c9af84c23b1847f1f5f894%40group.calendar.google.com/public/basic.ics][iCal link]]
+
+** Call for contribution
+
+This is a new section for particularly easy or interesting issues we would like folks external
+to the project to contribute to. We are available for mentoring and guiding you on their
+resolution. This is a great way to start making your mark on a complex project such as this
+one and to learn a lot in the process!
+
+- [[https://github.com/rust-gcc/gccrs/issues/2154][Issue 2154]]
+
+Derive macros (invoked through the ~#[derive(...)]~ attribute) should only be
+applied on type declarations - new structures, enumerations or unions. We need
+to report an error to the user when they try using such a macro on a non-type-declaration element - such as a function or trait.
+
+The provided playground link showcases the expected behavior. The code it contains will need to be added as a test case to the project, with other test cases welcome.
```

Comment:
```suggestion
The provided playground link (see [[https://github.com/rust-gcc/gccrs/issues/2154][the issue]])  showcases the expected behavior. The code it contains will need to be added as a test case to the project, with other test cases welcome.
```
very small nit: I was searching for a playground link in the report, but it's in the issue.